.github/actions/release-image/action.yml

```diff
@@ -291,16 +291,9 @@ runs:
         scanner: grype
         docker-config: .docker-rumble
 
-    - name: Setup chainctl
-      uses: chainguard-dev/actions/setup-chainctl@main
-      if: inputs.chainguardIdentity != ''
+    - uses: chainguard-dev/actions/setup-chainctl@main
```

Comment:
still needs `if: inputs.chainguardIdentity != ''` since this being used in other places

Reply:
Not just the two?

Reply:
I added it back, but we should sort out what changes we need to make this our default path.

Reply:
it is at least also used here: https://github.com/wolfi-dev/tools/blob/main/.github/workflows/release.yaml#L86